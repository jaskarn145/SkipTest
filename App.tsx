/**
 * Sample React Native App
 * https://github.com/facebook/react-native
 *
 * @format
 * @flow strict-local
 */

import React from 'react';
import {
  SafeAreaView
} from 'react-native';
import Header from './src/components/Header';
import DashBoard from './src/screens/DashBoard';

const App = () => {
  return (
    <SafeAreaView>
      <Header />
      <DashBoard />
    </SafeAreaView>
  )
}


export default App;
