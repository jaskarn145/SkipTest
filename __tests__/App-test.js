/**
 * @format
 * test to check the app component snapshot
 */

import 'react-native';
import React from 'react';
import App from '../App';

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer';

it('App Test', () => {
  let AppTest = renderer.create(<App />).toJSON();;
  expect(AppTest).toMatchSnapshot();
});
