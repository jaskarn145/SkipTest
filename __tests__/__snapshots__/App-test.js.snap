// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Test 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
>
  <View
    style={
      Object {
        "backgroundColor": "#d9a116",
        "fontSize": 10,
        "justifyContent": "center",
        "padding": 5,
        "width": "100%",
      }
    }
  >
    <Text
      style={
        Object {
          "color": "#f7f0f0",
          "fontSize": 30,
          "fontWeight": "bold",
          "paddingVertical": 10,
          "textAlign": "center",
          "width": "100%",
        }
      }
    >
      CODE TEST
    </Text>
  </View>
  <View>
    <View
      style={
        Object {
          "backgroundColor": "#ecf0f1",
          "flex": 1,
          "justifyContent": "center",
          "textAlign": "center",
        }
      }
    >
      <Modal
        animationType="none"
        hardwareAccelerated={false}
        onRequestClose={[Function]}
        statusBarTranslucent={true}
        supportedOrientations={
          Array [
            "landscape",
            "portrait",
          ]
        }
        transparent={true}
        visible={false}
      >
        <View
          style={
            Array [
              Object {
                "backgroundColor": "transparent",
                "bottom": 0,
                "flex": 1,
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              },
              Object {
                "backgroundColor": "rgba(0, 0, 0, 0.25)",
              },
            ]
          }
        >
          <View
            style={
              Object {
                "alignItems": "center",
                "bottom": 0,
                "justifyContent": "center",
                "left": 0,
                "position": "absolute",
                "right": 0,
                "top": 0,
              }
            }
          >
            <ActivityIndicator
              animating={true}
              color="white"
              hidesWhenStopped={true}
              size="large"
              style={
                Array [
                  Object {
                    "flex": 1,
                  },
                  Object {},
                ]
              }
            />
            <View
              style={
                Array [
                  Object {
                    "alignItems": "center",
                    "bottom": 0,
                    "flex": 1,
                    "justifyContent": "center",
                    "left": 0,
                    "position": "absolute",
                    "right": 0,
                    "top": 0,
                  },
                  Object {},
                ]
              }
            >
              <Text
                style={
                  Array [
                    Object {
                      "fontSize": 20,
                      "fontWeight": "bold",
                      "height": 50,
                      "top": 80,
                    },
                    Object {
                      "color": "#FFF",
                    },
                  ]
                }
              >
                Fetching Data...
              </Text>
            </View>
          </View>
        </View>
      </Modal>
    </View>
    <TextInput
      allowFontScaling={true}
      clearButtonMode="while-editing"
      defaultValue=""
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Search Here..."
      rejectResponderTermination={true}
      returnKeyType="search"
      style={
        Object {
          "backgroundColor": "#f7f0f0",
          "borderColor": "black",
          "borderRadius": 30,
          "borderWidth": 1,
          "fontSize": 15,
          "marginHorizontal": 10,
          "marginVertical": 10,
          "padding": 5,
          "paddingLeft": 10,
          "textTransform": "uppercase",
          "width": "95%",
        }
      }
      testID="SearchBar"
      underlineColorAndroid="transparent"
    />
    <View>
      <RCTScrollView
        data={Array []}
        disableVirtualization={false}
        getItem={[Function]}
        getItemCount={[Function]}
        horizontal={false}
        initialNumToRender={10}
        keyExtractor={[Function]}
        maxToRenderPerBatch={10}
        onContentSizeChange={[Function]}
        onEndReachedThreshold={0.5}
        onLayout={[Function]}
        onMomentumScrollEnd={[Function]}
        onScroll={[Function]}
        onScrollBeginDrag={[Function]}
        onScrollEndDrag={[Function]}
        removeClippedSubviews={false}
        renderItem={[Function]}
        scrollEventThrottle={400}
        stickyHeaderIndices={Array []}
        updateCellsBatchingPeriod={50}
        viewabilityConfigCallbackPairs={Array []}
        windowSize={21}
      >
        <View />
      </RCTScrollView>
    </View>
  </View>
</RCTSafeAreaView>
`;
